### ISSUE TITLE
Question on 'gradient_accumulation_steps'

### ISSUE THREAD

**Issue by sysu-zjw** [facebookresearch/DPR#201]
Hi, 

If I fine-tune DPR on two devices with parameters like below, is DPR supposed to give the same result?:
```
a) batch size=32, gpu_num=8, gradient_accumulation_steps=1
b) batch size=32, gpu_num=4, gradient_accumulation_steps=2
```
My concern is whether `gradient_accumulation_steps` can help handle a larger in-batch negative amount even on a smaller GPU device (kind of like a memory bank). While I don't see any related function on handling this, I conjecture it couldn't. 

Anyway, I think it would be better to confirm this with you. Thanks.

**Comment by vlad-karpukhin** [comment]
Hi @sysu-zjw ,
I haven't done any experiments for to compensate batch size with gradient accumulation so I don't know how effective it is for DPR model.

**Comment by sysu-zjw** [comment]
Thanks, @vlad-karpukhin .
